app/assets/javascripts/utilities/timeAgo.js

```diff
@@ -25,15 +25,34 @@ function secondsToHumanUnitAgo(seconds) {
   return wholeUnits + ' ' + unitName + ' ago';
 }
 
-function timeAgo(oldTimeInSeconds, maxDisplayedAge = 60 * 60 * 24 - 1) {
+/**
+ * Returns a given time in seconds as a human readable form, e.g. (5 min ago)
+ *
+ * @param {object} options
+ * @param {number} options.oldTimeInSeconds
+ * @param {function} [(humanTime) =>
+      `<span class="time-ago-indicator">(${humanTime})</span>`] options.formatter
+ * @param {number} [60 * 60 * 24 - 1] options.maxDisplayedAge The maximum display age in seconds
+ *
+ * @returns {string} A formatted string in human readable form. Note that the default formatter returns a string with markup in it.
+ */
+function timeAgo({
```

Comment:
For those unaware, if we start adding JS Docs documentation to our frontend code, it will provide Intellisense in VS Code. (Perhaps other editors as well, I don't know).

![image](https://user-images.githubusercontent.com/833231/74569977-a81e8200-4f49-11ea-9c25-031d7e59a4ed.png)

Reply:
Is there a way to generate that blob or did you hand type this?

Reply:
It generates a chunk when you type `/**` followed by enter in VS Code.